sql.go

```diff
@@ -145,17 +145,17 @@ func (s *sql) UpdatePrefix(prefix Prefix) (Prefix, error) {
 	if rows == 0 {
 		err := tx.Rollback()
 		if err != nil {
-			return Prefix{}, newOptimisticLockError("updatePrefix did not effect any row, but rollback did not work:" + err.Error())
+			return Prefix{}, fmt.Errorf("updatePrefix did not effect any row, but rollback did not work:%w", err)
```

Comment:
This changes the contract, because now the error is no longer the OptimisticLockError but the Error seen at Rollback. The client should really see the OptimisticLockError in my opinion.
Could even remove the rollback, see above.

Reply:
same